InstantSearch.js/routing-seo-friendly/src/search-router.js

```diff
@@ -0,0 +1,101 @@
+/* global instantsearch */
+
+function getCategorySlug(name) {
+  return name.replace(/\s/g, '+');
```

Comment:
We only "slugify" the spaces but what about punctation? It's maybe safer to rely on a third party.

> https://yjojj4lnrz.codesandbox.io/search/Ranges,+Cooktops+&+Ovens/?query=apple

Reply:
I think the problem for those is that if you remove multiple things by the same, the conversion is lossy. Those issues should be mostly solved by encodeURIComponent I think

Reply:
We could still encode/decode the category instead of manually do it. It will however be less readable.

What do we value the most?

Reply:
We can check on other websites how it's implemented. The safe bet is to use the encoded version. But we can mix both solutions: apply `encodeURIComponent` to each part of the value then apply the slug. For most cases, the encoding will be a noop which means that we have a readable URL.

Reply:
In 507ee489e233c8365e3b70168da782e26b96f034 I slugified the category and encoded/decoded.